external-crates/move/crates/move-compiler/src/cfgir/visitor.rs

```diff
@@ -773,37 +773,95 @@ impl<V: SimpleAbsIntConstructor> AbstractInterpreterVisitor for V {
 // utils
 //**************************************************************************************************
 
+pub fn has_special<FCommand, FExp>(
```

Comment:
Instead of `special`, maybe `any`? The names `has_any_command` and `has_any_exp` might better-communicate what this code is doing.